lib/node_modules/@stdlib/stats/base/dists/normal/quantile/benchmark/c/benchmark.c

```diff
@@ -93,13 +93,13 @@ static double random_uniform( const double min, const double max ) {
 * @return elapsed time in seconds
 */
 static double benchmark( void ) {
-	double elapsed;
-	double p[ 100 ];
-	double mu[ 100 ];
-	double sigma[ 100 ];
+	int i;	
 	double y;
 	double t;
-	int i;
+	double elapsed;
+	double p[100];
+	double mu[100];
+	double sigma[100];
```

Comment:
Variables should be ordered by length, in decreasing order. See other files for the expected ordering.